### PR TITLE
fix: security-secretstore-setup volume init semantics

### PR DESCRIPTION
**Pull request by vli11**
fixes: #3852
Signed-off-by: Valina Li <valina.li@intel.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?) NA; because it is a fix to a workaround
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?) NA; because it is a fix to a workaround
  <link to docs PR>

### CODE REVIEW

**Review by bnevis-i — CHANGES_REQUESTED** [review]
Both of these commands should have been MOVED from the Dockerfile to the entrypoint script

    && mkdir -p /vault/config/assets \
    && chown -Rh 100:1000 /vault/

**Review by bnevis-i — APPROVED** [review]
LGTM

File: cmd/security-secretstore-setup/Dockerfile [hunk 1]
```diff
@@ -52,8 +52,6 @@ COPY --from=builder /edgex-go/cmd/security-secretstore-setup/security-secretstor
 # Setup the entry point script, create token dir, and assign perms
 COPY --from=builder /edgex-go/cmd/security-secretstore-setup/entrypoint.sh /usr/local/bin/
 RUN chmod +x /usr/local/bin/entrypoint.sh \
-    && ln -s /usr/local/bin/entrypoint.sh / \
-    && mkdir -p /vault/config/assets \
-    && chown -Rh 100:1000 /vault/    
+    && ln -s /usr/local/bin/entrypoint.sh /
```
Comment (jim-wang-intel) [inline cmd/security-secretstore-setup/Dockerfile:55]: Please update the comment in line 52 to fit the real implementation

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]

**Review by bnevis-i — APPROVED** [review]

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**